docker-compose.yml

```diff
@@ -1,18 +1,25 @@
----
```

Comment:
Also, I would like to suggest adding this back in the YAML file as this is considered a part of the YAML syntax. Moreover, I would also recommend updating the file from `docker-compose.yml` to `dev.docker-compose.yml` and for the `prod.docker-compose.yml` file I would recommend leaving it as it with the original `docker-compose.yml` name. I think this would be better. What do you say ? :slightly_smiling_face:

Reply:
Usually we have a docker-compose at the root of the project for development purpose.
And i'll remove the production one but that's up to you

Reply:
> Usually we have a docker-compose at the root of the project for development purpose. And i'll remove the production one but that's up to you

No actually, I am not suggesting you to remove the `prod.docker-compose.yml` file just renaming the files as I suggested above. Other than that, the development docker compose file and the development Dockerfile looks perfect :slightly_smiling_face: .

Reply:
I understood what you asked, i'm just telling you what are the usual practice.